resources/data/elephpants.json

```diff
@@ -359,7 +359,14 @@
             "sponsor": "PHP Conference Poland",
             "year": 2019,
             "image": "https://i.imgur.com/QIyckVs.jpg"
+        },
+        {
+            "id": 46,
+            "name": "CakeDC Fest",
```

Comment:
It's just `CakeFest`

Reply:
I think this one is just `CakeDC`, correct?

Reply:
Yes:
![CakeDC photo](https://live.staticflickr.com/65535/48796213323_1efccd304c_z.jpg)